src/components/activity-page/managed-interactive/iframe-runtime.tsx

```diff
@@ -20,10 +20,11 @@ interface IProps {
   proposedHeight?: number;
   containerWidth?: number;
   setNewHint: (newHint: string) => void;
+  toggleModal?: any;
```

Comment:
We could probably type this with something like `toggleModal?: (modalProps: any) => void;` although then the `toggleModal` call below needs protection.